src/main/java/org/folio/service/manager/input/InputDataManagerImpl.java

```diff
@@ -249,12 +249,13 @@ private ExportPayload createExportPayload(OkapiConnectionParams okapiParams, Fil
     return exportPayload;
   }
 
-  private FileDefinition createExportFileDefinition(FileDefinition requestFileDefinition) {
-    String fileNameWithoutExtension = FilenameUtils.getBaseName(requestFileDefinition.getFileName());
+  private FileDefinition createExportFileDefinition(ExportRequest exportRequest) {
```

Comment:
This is what we are going to change in https://github.com/folio-org/mod-data-export/pull/42, Export request having FileDefinition is redundant, we should try not to use that